dart/test/sentry_traces_sampler_test.dart

```diff
@@ -80,6 +80,18 @@ void main() {
     expect(sut.sample(context).sampled, false);
   });
 
+  test('does not sample if tracesSampleRate and tracesSampleRate are null', () {
+    final sut = fixture.getSut(tracesSampleRate: null, tracesSampler: null);
+
+    final trContext = SentryTransactionContext(
+      'name',
+      'op',
+    );
+    final context = SentrySamplingContext(trContext, {});
+
+    expect(sut.sample(context).sampled, false);
```

Comment:
Previous test also had `expect(samplingDecision.sampleRate, isNull);`. Ignore if test without this is expected.

Reply:
added it